index.html

```diff
@@ -40,32 +41,277 @@ <h1 class="main-header">I’m Suliman Joya Glad to see you!</h1>
+              </ul>
+            </div>
+            <div class="btn">
+              <div class="buttons-wrapper">
+                <button type="button" class="buttons">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="all-projects">
+          <div class="project-image">
+            <img src="./images/project-3.svg" alt="project 3 image" />
+          </div>
+          <div class="project-info">
+            <div class="h1-wrapper">
+              <h2 class="h1-heading">Multi-Post Stories</h2>
+              <ul class="info">
+                <li class="company">CANOPY</li>
+                <li class="position">Back End Dev</li>
+                <li class="date">2015</li>
+              </ul>
+            </div>
+            <div>
+              <p class="paragraph">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="web-lang">
+                <li class="HTML">html</li>
+                <li class="CSS">css</li>
+                <li class="javaScript">javaScript</li>
+              </ul>
+            </div>
+            <div class="btn">
+              <div class="buttons-wrapper">
+                <button type="button" class="buttons">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="all-projects">
+          <div class="project-image">
+            <img src="./images/project-4.svg" alt="project 4 image" />
+          </div>
+          <div class="project-info">
+            <div class="h1-wrapper">
+              <h2 class="h1-heading">Multi-Post Stories</h2>
+              <ul class="info">
+                <li class="company">CANOPY</li>
+                <li class="position">Back End Dev</li>
+                <li class="date">2015</li>
+              </ul>
+            </div>
+            <div>
+              <p class="paragraph">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="web-lang">
+                <li class="HTML">html</li>
+                <li class="CSS">css</li>
+                <li class="javaScript">javaScript</li>
+              </ul>
+            </div>
+            <div class="btn">
+              <div class="buttons-wrapper">
+                <button type="button" class="buttons">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- The project pages are end here -->
+      <!-- ************************************ -->
+      <!-- The about page start here -->
+      <div class="about-page">
+        <section class="about-wrapper">
+          <h1 class="about-header">About Myself</h1>
```

Comment:
[OPTIONAL] Kindly change this to any other `hx` tag `form h2 to h6`. You can only have one `<h1>` tag for browsers SEO optimisation and for use by screen readers. [read here](https://ahrefs.com/blog/h1-tag/#:~:text=skim%20and%20digest.-,H1%20tags%20improve%20accessibility,title%20be%20an%20H1%20tag.)